server/@types/viewModels/index.d.ts

```diff
@@ -8,6 +8,8 @@ declare module 'viewModels' {
     receptionDate?: Date
     dateOfBirth?: Date
     location: string
+    restrictedPatient: boolean
```

Comment:
We needed to add and populate 2 new properties in our `PrisonerSummary` view model. These are the same property names that HMPPS Prisoner Profile uses, and indeed are the same property names in the returned `Prisoner` object from `prisoner-search-api` (so mapping is easy etc 👍 )